content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -83,4 +83,5 @@
     <filter root="/etc/workflow/instances/rep:policy"/>
     <filter root="/home/users/rep:policy"/>
     <filter root="/var/acs-commons/rep:policy"/>
+    <filter root="/apps/dam/gui/content/assets"/>
```

Comment:
This feels too intrusive to include by default. How can we make this an "opt-in" feature?

Reply:
I think then we can create a feature flag for this and write a render condition to show/hide this button based on the feature flag. 
https://sling.apache.org/documentation/the-sling-engine/featureflags.html 

What do you think ? But I will have to overlay the nodes from libs to add the button in the properties UI.

Reply:
I like the feature flag approach, but this is a very broad filter. It should be as narrow as possible to avoid conflicting with other customizations.